packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -520,6 +562,22 @@ impl PostgresDatabase {
         Ok(ids)
     }
 
+    async fn link_type_uris_to_version_ids<'p, I>(
```

Comment:
The lifetime `'p` is misleading (applies to `data_type_reference_ids` as well) as there is no parameter matching the name. Originally, `'p` was used for `PropertyType`, but then it got copy pasted (by me) without changing it.

I don't think it's worth changing now as I will merge these functions in a follow-up.